tests/execution/caching.md

```diff
@@ -7,8 +7,8 @@ schema @upstream(baseURL: "http://example.com", batch: {delay: 1, maxSize: 1000}
   query: Query
 }
 
-type Query @cache(maxAge: 100) {
-  bars: [Bar] @http(path: "/bars")
+type Query {
+  bars: [Bar] @http(path: "/bars") @cache(maxAge: 100)
```

Comment:
Is this test change required?

Reply:
otherwise cache doesn't work.

schema is defined only of fields https://github.com/tailcallhq/tailcall/blob/87130ecd94bc94bf197b4c896d9a88b53c6ed7c7/generated/.tailcallrc.graphql#L19-L25

and field.cache only is used:
https://github.com/tailcallhq/tailcall/blob/9997e824deaab0ca53f287848e2e14c5e54c0805/src/blueprint/definitions.rs#L348-L350

although it could be set to type 
https://github.com/tailcallhq/tailcall/blob/87130ecd94bc94bf197b4c896d9a88b53c6ed7c7/src/config/config.rs#L247

🤷🏻

Reply:
https://github.com/tailcallhq/tailcall/pull/1282

Reply:
This is fixed now. Let's revert the test.